packages/@n8n/task-runner/src/task-runner.ts

```diff
@@ -287,6 +305,8 @@ export abstract class TaskRunner {
 			data,
 		});
 		this.runningTasks.delete(taskId);
+		console.log(`Runner completed task ${taskId}`);
```

Comment:
This can become quite verbose. Since we don't have the notion of debug logs in the runner, could we omit this for now?

Reply:
Decided to remove these two logs. Let's add them in a dedicated PR if/when we need them.